change/@microsoft-teams-js-20f1c31d-4466-4278-b140-f9454214c18b.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "minor",
+  "comment": {
```

Comment:
This should be filled, otherwise Changelog won't have information on this change.

Reply:
Thank you, updated! Any preference with title/value? I didn't see other change files with both of these... just comment as a string.

This object is what was generated after the yarn changefile command